### PR TITLE
Pass on Tiny Items capable of being hidden in shoes + Clown can now hide items in his oversized shoes.

### PR DESCRIPTION
**Pull request by Yantarii**
**Following Items can now be hidden in shoes -**

- Spy Bugs (Syndi-Bug Kit) - Steal someones shoes and give em back
- Lighters
- Coins
- Vials
- Scalpels (Bringing in line w/ Knives.)
- Ciggarettes
- Pills
- Syndicate Shruikens.

Clown can now have items in his shoes.

### CODE REVIEW

File: code/game/objects/items/weapons/storage/fancy/vials.dm
Comment (CPU-Blanc) [inline code/game/objects/items/weapons/storage/fancy/vials.dm:1]: A whole.. box of vials? Seems a bit big to put in your shoes, no?

Comment (Yantarii) [reply code/game/objects/items/weapons/storage/fancy/vials.dm:1]: oops haha

**Comment by Yantarii** [comment]
Fixed.

**Review by CPU-Blanc — APPROVED** [review]
LGTM

**Pull request merged**